src/index.html

```diff
@@ -9,6 +9,38 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <form action="#"
+    method="post"
+    data-qa="big">
+      <div class="conteiner-big">
+        <label for="big-place-holder">
+          <img class="search-image-big" src="images/Search.svg" alt="search-image-big">
+          <input class="search-big"
+          type="text"
+          name="search-big"
+          id="search-big"
```

Comment:
you can remove `id` from input and `for` from the label because you write input inside the label